src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -253,7 +253,7 @@ SHOULD be present:
 | reference          | OPTIONAL              | Specification of the reference (e.g., 'mastoid', 'ElectrodeName01', 'intracranial', 'CAR', 'other', 'n/a'). If the channel is not an electrode channel (e.g., a microphone channel) use `n/a`.                                                                             |
 | group              | OPTIONAL              | Which group of channels (grid/strip/seeg/depth) this channel belongs to. This is relevant because one group has one cable-bundle and noise can be shared. This can be a name or number. Note that any groups specified in `_electrodes.tsv` must match those present here. |
 | sampling_frequency | OPTIONAL              | Sampling rate of the channel in Hz.                                                                                                                                                                                                                                        |
-| description        | OPTIONAL              | Brief free-text description of the channel, or other information of interest (e.g., position (e.g., "left lateral temporal surface", etc.).                                                                                                                                |
+| description        | OPTIONAL              | Brief free-text description of the channel, or other information of interest (e.g., position (e.g., "left lateral temporal surface")).                                                                                                                                |
```

Comment:
is it me and my sleep deprivation, but aren't these nested parenthesis hard to parse for the average human being?

Reply:
I agree with you, and I just had 8 hours of sound sleep :-) 

should be easy to avoid nested brackets in this case

Reply:
I think that I had a similar thought at one point and tried switching to square brackets nested in the parentheses, but that doesn't play well with markdown (or maybe one of the other formatters that the specification uses).